src/less/border.less

```diff
@@ -0,0 +1,6 @@
+@borderWidthSmall: .0625rem; // 1px
+@borderWidthMedium: .125rem; // 2px
+@borderWidthLarge: .1875rem; // 3px
+
+@borderRadiusSmall: .125rem; // 2px
+@borderRadiusMedium: .1875rem; // 3px
```

Comment:
ah, you mean generate classes for `border--top--s`, `borderRadius--topLeft--m`, etc?
that could be useful. i'll take a look and add a commit